### DISCUSSION TITLE
如何实现窗口最小化时暂停？

### DISCUSSION THREAD

**Discussion by Aurora-Simba** [hooke007/mpv_PlayKit#381] [Q&A 问答] [answered]
如题

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/blob/054faa9571ab4d1580f977ca7fc01b902ca6adf7/portable_config/profiles.conf#L21-L25

参考profile如何写的，一共涉及两个要素
最小化的属性 https://mpv.io/manual/master/#options-window-minimized
暂停的选项 https://mpv.io/manual/master/#options-pause

**Comment by zhongfly** [comment, marked as answer]
```
[minimized]
 profile-desc=最小化时暂停
 profile-cond=window_minimized and vid and not p["current-tracks/video/albumart"]
 profile-restore=copy-equal
 pause
```

**Comment by nathalie-annis** [comment]
能实现窗口失去焦点时暂停吗?
谢谢大佬指点,已经实现:
[minimized]
 profile-desc=窗口失焦时暂停
 profile-cond=not focused and vid and not p["current-tracks/video/albumart"]
 profile-restore=copy-equal
 pause

**Reply by hooke007** [reply]
https://mpv.io/manual/master/#command-interface-focused
